telemetry/definitions/commonDefinitions.json

```diff
@@ -3818,14 +3857,6 @@
             "passive": true,
             "description": "Deprecated - avoid logging every service call",
             "metadata": [
-                {
-                    "type": "codeTransformApiErrorMessage",
-                    "required": true
-                },
```

Comment:
are you sure this wont break any implementation?

Reply:
None of the other metric types make use of this field, and it was recommended by Chris to make use of one of the default fields in place of this one. But I can leave it if we'd rather not risk it.